app/src/main/java/io/github/boogiemonster1o1/eyeyoureadyforit/App.java

```diff
@@ -124,6 +124,7 @@ public static void main(String[] args) {
 				.filter(event -> event.getMessage().getContent().startsWith("!eyeyoureadyforit "))
 				.subscribe(event -> {
 					if (event.getMessage().getContent().equals("!eyeyoureadyforit reload")) {
+						LOGGER.info("Reloading data...");
```

Comment:
CONCERN NO SEND MESSAGE IN DISCORD AS WELL OR BAD

Reply:
dun